app/models/survey/answer.rb

```diff
@@ -21,6 +23,9 @@ class Survey::Answer < ActiveRecord::Base
   before_save :check_single_choice_with_field_case
 
   def value
+    # TODO: number type option should calculate the value based on custom formula
```

Comment:
is this related to https://offcents.atlassian.net/browse/OF-1198?

Reply:
Yes, exactly!